content/en/docs/concepts/storage/volumes.md

```diff
@@ -1291,8 +1291,11 @@ Once a CSI compatible volume driver is deployed on a Kubernetes cluster, users
 may use the `csi` volume type to attach, mount, etc. the volumes exposed by the
 CSI driver.
 
-The `csi` volume type does not support direct reference from Pod and may only be
-referenced in a Pod via a `PersistentVolumeClaim` object.
+A `csi` volume can be used in a pod in three different ways:
+- through a reference to a [`persistentVolumeClaim`](#persistentvolumeclaim)
+- with a [generic ephemeral volume](/docs/concepts/storage/ephemeral-volumes/)
```

Comment:
This looks like a chunk that should be in generic ephemeral volume PR. And generic ephemeral volumes are not limited to CSI, in-tree volumes can be used too.

Reply:
Hmm, right. Removed.